common/tools/dev-tool/src/commands/samples/index.ts

```diff
@@ -9,5 +9,6 @@ export default subCommand(commandInfo, {
   dev: () => import("./dev"),
   prep: () => import("./prep"),
   run: () => import("./run"),
-  "ts-to-js": () => import("./tsToJs")
+  "ts-to-js": () => import("./tsToJs"),
+  checkNodesVers: () => import("./checkNodeVers")
```

Comment:
By convention this should be `"check-node-versions"`. I don't see many CLI programs use camelCase verbs.

Reply:
How about something more about how the command is meant to be used: `"check-release"`

Reply:
There could be uses for this other than for checking for release so I am hesitant on renaming to `check-release`. I will go with `check-node-versions`.